doc/sphinx/README.rst

```diff
@@ -202,6 +202,9 @@ In addition to the objects above, the ``coqrst`` Sphinx plugin defines the follo
           Print nat.
           Definition a := 1.
 
+    The blank line after the directive is required.  If you begin a proof,
+    include an ``Abort`` afterwards to reset coqtop for the next example.
```

Comment:
You also need to modify the file this one is generated from (see comments at top).

Reply:
Don't forget about this.

Reply:
Sorry you had already taken this comment into account.